infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/limits/ram/RamLimitProvisioner.java

```diff
@@ -39,7 +39,10 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
       for (Container container : pod.getSpec().getContainers()) {
         final Map<String, String> attributes =
             machines.get(machineName(pod, container)).getAttributes();
-        Containers.addRamLimit(container, Long.parseLong(attributes.get(MEMORY_LIMIT_ATTRIBUTE)));
+        String memoryLimitAttribute = attributes.get(MEMORY_LIMIT_ATTRIBUTE);
+        if (memoryLimitAttribute != null) {
+          Containers.addRamLimit(container, Long.parseLong(memoryLimitAttribute));
```

Comment:
It's like an unusual case since we have configuration properties for configuring default values for machines memory if they are not specified explicitly. Should it be logged as a warning?

Reply:
Makes sense. Ok to me leave it as it is